### DISCUSSION TITLE
translation sync warning message is displayed in wagtail admin - what to do next?

### DISCUSSION THREAD

**Discussion by leonh** [wagtail/wagtail-localize#370] [Q&A] [answered]
I have added a field to a model that is being translated .

On the wagtail edit page I can see a warning

"The data model for 'Country location page' has been changed since the last translation sync. If any new fields have been added recently, these may not be visible until the next translation sync."

How do I proceed from here ?  
I can't find anything in the docs...

I tried running `./manage.py sync_local_trees` but the warning message remains.

Does anyone know why this is happening ?

**Comment by kaedroho** [comment, marked as answer]
"translation sync" is referring to an editor either submitting a page for translation for the first time or using the "sync translated pages" feature.

This warning occurs when the model itself is changed since the last time that particular page was synced by an editor, so the translation may be missing fields that are on the model. The solution is to find the original language version of the page in the explorer and click "sync translated pages" in the "more" menu.

This could perhaps be improved by adding a button to the message linking the user straight to that interface.

**Reply by leonh** [reply]
Yes that would be useful.
